config/dev.json

```diff
@@ -1,9 +1,9 @@
 {
-    "Environment": {
-      "baseUrl": "https://www.amazon.in"
-    },
-    "TestData1": {
+  "Environment": {
+    "baseUrl": "https://www.amazon.in/"
+  },
```

Comment:
remove slash at the end
usage for child classes should be like this,
ex: open(/search)